build.gradle

```diff
@@ -57,8 +54,8 @@ sourceSets {
 }
 
 dependencies {
-    shadow gradleApi()
-    shadow localGroovy()
+    implementation gradleApi()
```

Comment:
gradleApi() is automatically added by java-gradle-plugin, and not required here.  If we do with to leave it, it should be 'compileOnly' as the target gradle install (where the plugin is applied) provides this.

Reply:
thanks for pointing this out!